api/db/psqlretriever.go

```diff
@@ -1,9 +1,39 @@
 package db
 
 import (
+	"fmt"
+	"regexp"
+	"strings"
 	"time"
+
+	"github.com/lib/pq"
 )
 
+// ConvertStringToSlice parses the output string from the array type.
+// Regex used: (((?P<value>(([^",\\{}\s(NULL)])+|"([^"\\]|\\"|\\\\)*")))(,)?)
+func ConvertStringToSlice(array string) []string {
```

Comment:
It makes sense if this function be inside our `util` package?

Reply:
Yes, it could make sense. This can be made in a refactoring. Do you mind to open a small issue since this change doesn't affect the current feature?